README.md

```diff
@@ -35,21 +35,98 @@ been granted access to some resource. It has similarities with authorization flo
+* __OCM Address__ - a string of the form `<Receiving Party's identifier>@<fqdn>` which can be used to uniquely identify a user or group "at" an OCM-capable server. `<Receiving Party's identifier>` is an opaque string,
+unique at the server. `<fqdn>` is the Fully Qualified Domain Name by which the server is identified. This can, but doesn't need to be, the domain at which the OCM API of that server is hosted.
+* __OCM Notification__ - a message from the Receiving Server to the Sending Server or vice versa, using the OCM Notifications endpoint.
+* __Invite Message__ - out-of-band message used to establish contact between parties and servers in the Invite Flow, containing an Invite Token (see below) and the Invite Sender's OCM Address
+* __Invite Sender__ - the party sending an Invite
+* __Invite Receiver__ - the party receiving an Invite
+* __Invite Sender OCM Server__ - the server holding an addressbook used by the Invite Sender, to which details of the Invite Receiver are to be added
+* __Invite Receiver OCM Server__ - the server holding an addressbook used by the Invite Receiver, to which details of the Invite Sender are to be added
+* __Invite Token__ - a hard-to-guess string used in the Invite Flow, generated by the Invite Sender OCM Server and linked uniquely to the Invite Sender's OCM Address
+* __Invite Creation Gesture__ - gesture from the Invite Sender to the Invite Sender OCM Server, resulting in the creation of an Invite Token.
+* __Invite Acceptance Gesture__ - gesture from the Invite Receiver to the Invite Receiver OCM Server, supplying the Invite Token as well as the OCM Address of the Invite Sender, effectively whitelisting the Invite Sender OCM Server for sending Share Creation Notifications to the Invite Receiver OCM Server.
+* __Invite Acceptance Request__ - API call from the Invite Receiver OCM Server to the Invite Sender OCM Server, supplying the Invite Token as well as the OCM Address of the Invite Receiver, effectively whitelisting the Invite Sender OCM Server for sending Share Creation Notifications to the Invite Receiver OCM Server.
+* __Invite Acceptance Response__ - HTTP response to the Invite Acceptance Request
+
+### Establishing Contact
+Before the Sending Server can send a Share Creation Notification to the Receiving Server, it needs to establish the Receiving Server's FQDN, and the Receiving Party's identifier, among other things.
+Some steps may proceed the Sending Gesture, allowing the Sending Party to establish (with some level of trust) the OCM Address of the Receiving Party.
+
+#### Direct Entry
+The simplest way for this is if the Receiving Party shares their OCM Address with the Sending Party through some out-of-band means, and the Sending Party enters this string into the user interface of the Sending Server, by means of typing or pasting into an HTML form, or clicking a link to a URL that includes the string in some form.
+
+#### Addressbooks
```

Comment:
Minor nit pick: should be two words `Adress books` here, and below

Reply:
Thanks! Fixed